conans/client/generators/msbuild.py

```diff
@@ -38,7 +38,7 @@ class MSBuildGenerator(Generator):
             <conan_{name}_props_imported>True</conan_{name}_props_imported>
           </PropertyGroup>
           <PropertyGroup>
-            <LocalDebuggerEnvironment>PATH=%PATH%;$(Conan{name}BinaryDirectories)</LocalDebuggerEnvironment>
+            <LocalDebuggerEnvironment>PATH=%PATH%;$(Conan{name}BinaryDirectories)$(LocalDebuggerEnvironment)</LocalDebuggerEnvironment>
```

Comment:
no separator (semicolon)?

Reply:
Nop, that works, and it is the same pattern used for include-dirs and other properties, check the current implementation of ``MSBuildGenerator`` (I was initially surprised too)